content/tracing/setup/php.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Tracing PHP Applications
+kind: Documentation
+further_reading:
+- link: "https://github.com/DataDog/dd-trace-php"
+  tag: "Github"
+  text: Source code
+- link: "tracing/visualization/"
+  tag: "Documentation"
+  text: "Explore your services, resources and traces"
+---
+
+PHP APM is currently in **alpha**** and not officially supported by Datadog. The officially supported PHP Beta will launch as early as September 2018.  To be notified of the beta launch, [submit this form][1].
+
+The unstable alpha tracer can be [accessed today on Github][2].
+**We do not recommend running the alpha tracer in production.**
+
+## Automatic Instrumentation
+
+APM PHP support is currently in alpha and does not yet provide any automatic instrumentation.  Beta support is planned for September 2018 and will provide automatic instrumentation for popular frameworks and libraries.  See some of these listed below.  
```

Comment:
APM PHP support is currently in alpha and provides limited automatic instrumentation.  Planned beta support will provide automatic instrumentation for popular frameworks and libraries.  See some of these listed below.